src/substitutions.ts

```diff
@@ -34,37 +39,22 @@ interface IDisplayMap {
     [name: string]: string;
 }
 
-interface IFirstWordMap {
-    [firstword: string]: [IDisplayMap];
+interface PillItem {
+    id: string;
+    text: string;
 }
 
 export interface ISlackToMatrixResult {
```

Comment:
This interface feels like it's named backwards? It's only used in `matrixToSlack`?

Reply:
It is :(